src/AppBundle/Controller/DashboardController.php

```diff
@@ -0,0 +1,345 @@
+<?php
```

Comment:
You should have a look about [param converters](https://symfony.com/doc/2.8/best_practices/controllers.html#using-the-paramconverter).

What would happen if I give a wrong id in some routes? Errors are not handled.
Furthermore, you'll have the entity object directly in your method without querying for it by yourself, thus, less code to write.